core/src/toggle/toggle.tsx

```diff
@@ -0,0 +1,33 @@
+import styles from "./toggle.module.css";
+
+interface Props {
+	name?: string;
+	onChange?: (state: boolean) => void;
```

Comment:
sorry I know @ng-hai already requested this but I think it'd be even better (and make this compatible with form builders) by supporting both "onChange" (which is event-based) and "setValue" (which is value-based, for manual uses)